models/Media/PostPicture.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+const sharp = require('sharp');
+
+const PostPictureSchema = new Schema({
+	buffer: {
+		type: Buffer,
+		required: true,
+	},
+});
+
+PostPictureSchema.statics.create = function (buffer) {
+	return new Promise((resolve, reject) => {
+		sharp(buffer)
+		.resize({ height: 600, width: 600, withoutEnlargement: true })
```

Comment:
Why are we resizing? Does a non-square image cause issues in the UI?

Reply:
This was introduced as part of #86. Here I'm just moving the logic from the `post` model to the `postPicture` model.

I had very similar questions in that PR which were resolved over a private chat. Essentially though, it's just to reduce file size.

Although looking it at it here, I think this is supposed to be using the [`fit inside`](https://sharp.dimens.io/en/stable/api-resize/#resize) option in order to maintain aspect ratio...

@scholvat can you verify this? Its being used in your scripts but not in the actual code.

Reply:
Yep that’s a mistake. Feel free to fix it in this PR or make a separate patch to update that. It should match what is in the scripts.